threescale_api/auth.py

```diff
@@ -37,13 +36,12 @@ class UserKeyAuth(BaseClientAuth):
     "Provides user_key authentication for api client calls"
     @property
     def credentials(self):
-        return (
-            self.app.service.proxy.list()["auth_user_key"],
-            self.app["user_key"])
+        return {
+            self.app.service.proxy.list()["auth_user_key"]: self.app["user_key"]}
 
     def __call__(self, r):
         if self.location == "authorization":
-            auth = requests.auth.HTTPBasicAuth(self.credentials[1], "")
+            auth = requests.auth.HTTPBasicAuth(list(self.credentials.values())[0], "")
```

Comment:
I would suggest using another construct - in order to avoid creating a new instance of the `list`:
`next(iter(mydict.values()))`